lisa/operating_system.py

```diff
@@ -198,19 +236,46 @@ def name_pattern(cls) -> Pattern[str]:
     def _initialize_package_installation(self) -> None:
         self._node.execute("apt-get update", sudo=True)
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> ExecutableResult:
         command = (
             f"DEBIAN_FRONTEND=noninteractive "
             f"apt-get -y install {' '.join(packages)}"
         )
-        self._node.execute(command, sudo=True)
+        if not signed:
+            command = command.__add__(" --allow-unauthenticated")
+
+        return self._node.execute(command, sudo=True)
+
+    def _get_os_version(self) -> OsVersion:
+        os_version = OsVersion("")
+        cmd_result = self._node.execute(cmd="which lsb_release", no_error_log=True)
+        if cmd_result.exit_code != 0 and cmd_result.stdout != "":
```

Comment:
Will it not exist? if so, throw an exception.

Reply:
> Will it not exist? if so, throw an exception.

my bad. It should be  "if cmd_result.exit_code == 0 and cmd_result.stdout != "":"

Reply:
I mean, can we assume it exists? Because the OS is determinated already.

Reply:
no we cannot assume. Because in some older versions, i don't think lsb_release exists.

Reply:
I cannot find those lines in latest code. But if this cmd doesn't exists, there should be an exception, we can fix in that distro. If we don't test that old distro, we don't need to handle it.